package.json

```
@@ -8,7 +8,8 @@
     "lint:fix": "tslint -p tsconfig.json",
     "build:tsc": "tsc -p tsconfig.json",
     "watch:build:tsc": "tsc -p tsconfig.json --watch",
-    "start": "npm run build:tsc && node dist/server.js"
+    "start": "npm run build:tsc && node dist/server.js",
+    "indexScenes": "tsc scripts/indexScenes.ts && node scripts/indexScenes.js"
```

`npm run build:tsc` already builds `scripts`:

```suggestion
    "indexScenes": "npm run build:tsc && node dist/scripts/indexScenes.js"
```